### PR TITLE
Support for admin interface

### PR DESCRIPTION
**Pull request by koushikkm**
This brings the admin support in general.
Aligned the user model to work with admin interface.
Brought admin support for blog also.

To test, you need to create a superuser from console: ./manage.py createsuperuser
Then runserver
then check localhost:8000/admin
user super user account for admin login

Note: 
1. Fresh migrations may be needed with database for testing.
2. Some changes have come from other branch as I merged them. Ideally, I should have done the admin support from a separate branch to avoid lots of already reviewed code in a code review.

### CODE REVIEW

**Comment by koushikkm** [comment]
@pankajssp

File: account/templates/account/password_reset_complete.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends "account/thome.html" %}
```
Comment (koushikkm) [inline account/templates/account/password_reset_complete.html]: ignore this file, came from merge

File: account/templates/account/password_reset_confirm.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{% extends "account/thome.html" %}
```
Comment (koushikkm) [inline account/templates/account/password_reset_confirm.html]: ignore this file, came from merge

File: account/templates/account/password_reset_done.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% extends "account/thome.html" %}
```
Comment (koushikkm) [inline account/templates/account/password_reset_done.html]: ignore this file, came from merge

**Comment by pankajssp** [comment]
Looks good. Can be merged.

**Pull request merged**